sdk/test-utils/test-credential/CHANGELOG.md

```diff
@@ -10,6 +10,18 @@
 
 ### Other Changes
 
+## 1.3.1 (2024-09-09)
```

Comment:
we never port changelogs from hotfixes, but probably we should.

Reply:
Ya, I think we should be porting changelogs from hotfixes ideally - can't see a downside to omitting it other than it's easy to forget (which is not a valid reason 😄 )

But... happy to hear objections

Reply:
More than forgetting, I don't think we have ever added it to the guidelines even.

Reply:
So what do you think @HarshaNalluru - do you want me to merge this? I think it's good to have the CHANGELOG present the changes across branches, but I want to make sure I address any objections you have, if any. Are you good with me merging this?

Reply:
yes please merge